### ISSUE TITLE
"myapi": executable file not found in $PATH: unknown when using docker 

### ISSUE THREAD

**Issue by brianbolt** [karec/cookiecutter-flask-restful#45]
Seems to be failing when installing and using all the defaults and then using the `make init` command

**Error**

```
Recreating restful_api_web_1 ... error

ERROR: for restful_api_web_1  Cannot start service web: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "myapi": executable file not found in $PATH: unknown

ERROR: for web  Cannot start service web: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "myapi": executable file not found in $PATH: unknown
```

Full setup and init output below:

**Setup**

```
bbolt@bbolt playground % cookiecutter https://github.com/karec/cookiecutter-flask-restful
project_name [restful_api]: 
app_name [myapi]: 
Select python_version:
1 - 3.8
2 - 3.7
3 - 3.6
Choose from 1, 2, 3 [1]: 
tox_python_env [py38]: 
Select use_celery:
1 - no
2 - yes
Choose from 1, 2 [1]: 
admin_user_username [admin]: 
admin_user_email [admin@mail.com]: 
admin_user_password [admin]: 
Select wsgi_server:
1 - none
2 - uwsgi
3 - gunicorn
Choose from 1, 2, 3 [1]: 
```

**Init**

```
bbolt@bbolt playground % cd restful_api 
bbolt@bbolt restful_api % make init
docker-compose build
Building web
Step 1/10 : FROM python:3.8
 ---> b0358f6298cd
Step 2/10 : RUN mkdir /code
 ---> Using cache
 ---> 87c9c870b25e
Step 3/10 : WORKDIR /code
 ---> Using cache
 ---> f21a622260a4
Step 4/10 : COPY requirements.txt setup.py tox.ini ./
 ---> 1706cabbf3ef
Step 5/10 : RUN pip install -U pip
 ---> Running in abc8ae00d385
Requirement already satisfied: pip in /usr/local/lib/python3.8/site-packages (20.3.3)
Removing intermediate container abc8ae00d385
 ---> 6d7debcf7641
Step 6/10 : RUN pip install -r requirements.txt
 ---> Running in 45e00103d363
Collecting marshmallow>=3
  Downloading marshmallow-3.10.0-py2.py3-none-any.whl (46 kB)
Collecting apispec-webframeworks
  Downloading apispec_webframeworks-0.5.2-py2.py3-none-any.whl (12 kB)
Collecting apispec[yaml]
  Downloading apispec-4.0.0-py2.py3-none-any.whl (26 kB)
Collecting PyYAML>=3.10
  Downloading PyYAML-5.3.1.tar.gz (269 kB)
Collecting flask
  Downloading Flask-1.1.2-py2.py3-none-any.whl (94 kB)
Collecting click>=5.1
  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)
Collecting itsdangerous>=0.24
  Downloading itsdangerous-1.1.0-py2.py3-none-any.whl (16 kB)
Collecting Jinja2>=2.10.1
  Downloading Jinja2-2.11.2-py2.py3-none-any.whl (125 kB)
Collecting MarkupSafe>=0.23
  Downloading MarkupSafe-1.1.1-cp38-cp38-manylinux1_x86_64.whl (32 kB)
Collecting Werkzeug>=0.15
  Downloading Werkzeug-1.0.1-py2.py3-none-any.whl (298 kB)
Collecting flask-jwt-extended
  Downloading Flask-JWT-Extended-3.25.0.tar.gz (31 kB)
Collecting PyJWT<2.0,>=1.6.4
  Downloading PyJWT-1.7.1-py2.py3-none-any.whl (18 kB)
Collecting flask-marshmallow
  Downloading flask_marshmallow-0.14.0-py2.py3-none-any.whl (10 kB)
Collecting six
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting flask-migrate
  Downloading Flask_Migrate-2.5.3-py2.py3-none-any.whl (13 kB)
Collecting flask-sqlalchemy
  Downloading Flask_SQLAlchemy-2.4.4-py2.py3-none-any.whl (17 kB)
Collecting alembic>=0.7
  Downloading alembic-1.4.3-py2.py3-none-any.whl (159 kB)
Collecting python-editor>=0.3
  Downloading python_editor-1.0.4-py3-none-any.whl (4.9 kB)
Collecting SQLAlchemy>=0.8.0
  Downloading SQLAlchemy-1.3.22-cp38-cp38-manylinux2010_x86_64.whl (1.3 MB)
Collecting flask-restful
  Downloading Flask_RESTful-0.3.8-py2.py3-none-any.whl (25 kB)
Collecting aniso8601>=0.82
  Downloading aniso8601-8.1.0-py2.py3-none-any.whl (44 kB)
Collecting marshmallow-sqlalchemy
  Downloading marshmallow_sqlalchemy-0.24.1-py2.py3-none-any.whl (18 kB)
Collecting passlib
  Downloading passlib-1.7.4-py2.py3-none-any.whl (525 kB)
Collecting python-dotenv
  Downloading python_dotenv-0.15.0-py2.py3-none-any.whl (18 kB)
Collecting tox
  Downloading tox-3.21.0-py2.py3-none-any.whl (84 kB)
Collecting filelock>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting packaging>=14
  Downloading packaging-20.8-py2.py3-none-any.whl (39 kB)
Collecting pluggy>=0.12.0
  Downloading pluggy-0.13.1-py2.py3-none-any.whl (18 kB)
Collecting py>=1.4.17
  Downloading py-1.10.0-py2.py3-none-any.whl (97 kB)
Collecting pyparsing>=2.0.2
  Downloading pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)
Collecting toml>=0.9.4
  Downloading toml-0.10.2-py2.py3-none-any.whl (16 kB)
Collecting virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0
  Downloading virtualenv-20.3.0-py2.py3-none-any.whl (5.7 MB)
Collecting appdirs<2,>=1.4.3
  Downloading appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting distlib<1,>=0.3.1
  Downloading distlib-0.3.1-py2.py3-none-any.whl (335 kB)
Collecting Mako
  Downloading Mako-1.1.3-py2.py3-none-any.whl (75 kB)
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting pytz
  Downloading pytz-2020.5-py2.py3-none-any.whl (510 kB)
Building wheels for collected packages: PyYAML, flask-jwt-extended
  Building wheel for PyYAML (setup.py): started
  Building wheel for PyYAML (setup.py): finished with status 'done'
  Created wheel for PyYAML: filename=PyYAML-5.3.1-cp38-cp38-linux_x86_64.whl size=572488 sha256=5757dd393e674951a1bc399b335ae17843b17d8745797072a231b2452a416863
  Stored in directory: /root/.cache/pip/wheels/13/90/db/290ab3a34f2ef0b5a0f89235dc2d40fea83e77de84ed2dc05c
  Building wheel for flask-jwt-extended (setup.py): started
  Building wheel for flask-jwt-extended (setup.py): finished with status 'done'
  Created wheel for flask-jwt-extended: filename=Flask_JWT_Extended-3.25.0-py2.py3-none-any.whl size=21569 sha256=27dd3a35f6c4e6767fa4a06541c8499490d1a8489c77746d85ab54bf2a2cc89e
  Stored in directory: /root/.cache/pip/wheels/f0/53/54/ec3db050e12900460634d2f7badb842fa29af7508a5b84557d
Successfully built PyYAML flask-jwt-extended
Installing collected packages: MarkupSafe, Werkzeug, six, Jinja2, itsdangerous, click, SQLAlchemy, PyYAML, python-editor, python-dateutil, pyparsing, Mako, flask, filelock, distlib, appdirs, apispec, virtualenv, toml, pytz, PyJWT, py, pluggy, packaging, marshmallow, flask-sqlalchemy, aniso8601, alembic, tox, python-dotenv, passlib, marshmallow-sqlalchemy, flask-restful, flask-migrate, flask-marshmallow, flask-jwt-extended, apispec-webframeworks
Successfully installed Jinja2-2.11.2 Mako-1.1.3 MarkupSafe-1.1.1 PyJWT-1.7.1 PyYAML-5.3.1 SQLAlchemy-1.3.22 Werkzeug-1.0.1 alembic-1.4.3 aniso8601-8.1.0 apispec-4.0.0 apispec-webframeworks-0.5.2 appdirs-1.4.4 click-7.1.2 distlib-0.3.1 filelock-3.0.12 flask-1.1.2 flask-jwt-extended-3.25.0 flask-marshmallow-0.14.0 flask-migrate-2.5.3 flask-restful-0.3.8 flask-sqlalchemy-2.4.4 itsdangerous-1.1.0 marshmallow-3.10.0 marshmallow-sqlalchemy-0.24.1 packaging-20.8 passlib-1.7.4 pluggy-0.13.1 py-1.10.0 pyparsing-2.4.7 python-dateutil-2.8.1 python-dotenv-0.15.0 python-editor-1.0.4 pytz-2020.5 six-1.15.0 toml-0.10.2 tox-3.21.0 virtualenv-20.3.0
Removing intermediate container 45e00103d363
 ---> 30025f46750e
Step 7/10 : RUN pip install -e .
 ---> Running in f0ab77a80cb4
Obtaining file:///code
Requirement already satisfied: flask in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (1.1.2)
Requirement already satisfied: flask-sqlalchemy in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (2.4.4)
Requirement already satisfied: flask-restful in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (0.3.8)
Requirement already satisfied: flask-migrate in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (2.5.3)
Requirement already satisfied: flask-jwt-extended in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (3.25.0)
Requirement already satisfied: flask-marshmallow in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (0.14.0)
Requirement already satisfied: marshmallow-sqlalchemy in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (0.24.1)
Requirement already satisfied: python-dotenv in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (0.15.0)
Requirement already satisfied: passlib in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (1.7.4)
Requirement already satisfied: apispec[yaml] in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (4.0.0)
Requirement already satisfied: apispec-webframeworks in /usr/local/lib/python3.8/site-packages (from myapi==0.1) (0.5.2)
Requirement already satisfied: PyYAML>=3.10 in /usr/local/lib/python3.8/site-packages (from apispec[yaml]->myapi==0.1) (5.3.1)
Requirement already satisfied: click>=5.1 in /usr/local/lib/python3.8/site-packages (from flask->myapi==0.1) (7.1.2)
Requirement already satisfied: itsdangerous>=0.24 in /usr/local/lib/python3.8/site-packages (from flask->myapi==0.1) (1.1.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.8/site-packages (from flask->myapi==0.1) (2.11.2)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.8/site-packages (from flask->myapi==0.1) (1.0.1)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.8/site-packages (from Jinja2>=2.10.1->flask->myapi==0.1) (1.1.1)
Requirement already satisfied: PyJWT<2.0,>=1.6.4 in /usr/local/lib/python3.8/site-packages (from flask-jwt-extended->myapi==0.1) (1.7.1)
Requirement already satisfied: six in /usr/local/lib/python3.8/site-packages (from flask-jwt-extended->myapi==0.1) (1.15.0)
Requirement already satisfied: marshmallow>=2.0.0 in /usr/local/lib/python3.8/site-packages (from flask-marshmallow->myapi==0.1) (3.10.0)
Requirement already satisfied: alembic>=0.7 in /usr/local/lib/python3.8/site-packages (from flask-migrate->myapi==0.1) (1.4.3)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.8/site-packages (from alembic>=0.7->flask-migrate->myapi==0.1) (2.8.1)
Requirement already satisfied: SQLAlchemy>=1.1.0 in /usr/local/lib/python3.8/site-packages (from alembic>=0.7->flask-migrate->myapi==0.1) (1.3.22)
Requirement already satisfied: python-editor>=0.3 in /usr/local/lib/python3.8/site-packages (from alembic>=0.7->flask-migrate->myapi==0.1) (1.0.4)
Requirement already satisfied: Mako in /usr/local/lib/python3.8/site-packages (from alembic>=0.7->flask-migrate->myapi==0.1) (1.1.3)
Requirement already satisfied: aniso8601>=0.82 in /usr/local/lib/python3.8/site-packages (from flask-restful->myapi==0.1) (8.1.0)
Requirement already satisfied: pytz in /usr/local/lib/python3.8/site-packages (from flask-restful->myapi==0.1) (2020.5)
Installing collected packages: myapi
  Running setup.py develop for myapi
Successfully installed myapi
Removing intermediate container f0ab77a80cb4
 ---> 7e90d7594f1f
Step 8/10 : COPY myapi myapi/
 ---> 60bef73528f1
Step 9/10 : COPY migrations migrations/
 ---> 2e2dbb23e90d
Step 10/10 : EXPOSE 5000
 ---> Running in e557f96dd4e0
Removing intermediate container e557f96dd4e0
 ---> 2fbe405e8216

Successfully built 2fbe405e8216
Successfully tagged myapi:latest
docker-compose up -d
Recreating restful_api_web_1 ... error

ERROR: for restful_api_web_1  Cannot start service web: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "myapi": executable file not found in $PATH: unknown

ERROR: for web  Cannot start service web: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "myapi": executable file not found in $PATH: unknown
ERROR: Encountered errors while bringing up the project.
make: *** [run] Error 1
```

**Comment by karec** [comment]
Hello @brianbolt and thanks for reporting.

This issue should now be solved with 955b0de

**Comment by brianbolt** [comment]
@karec Thanks for the quick fix!
